### DISCUSSION TITLE
Copy static files to project root directory from nuget package

### DISCUSSION THREAD

**Discussion by kuntalbose** [NuGet/Home#11275] [Q&A]
I am trying to build a nuget package in .Net Core for that I create a .net core class
Library project and also add 2  json config files for configuration which I also 
add into package .

I want ,when I add the package into my project then the 2 json files placed into project folder root
or where I set the path.

for this I do following steps ,but not working,**files added as a link** ,my **requirement
is add files physically into directory**
 

[![enter image description here][1]][1]

**Create a class library project**

[![enter image description here][2]][2]

**Set JSON file property**

[![enter image description here][3]][3]

**Set package options**

[![enter image description here][4]][4]

**after adding the package ,json files add as link.**

I want a less complex straight forward solution.

Please help.

  [1]: https://i.stack.imgur.com/clDit.png
  [2]: https://i.stack.imgur.com/BbeMF.png
  [3]: https://i.stack.imgur.com/HkT2C.png
  [4]: https://i.stack.imgur.com/NuhQc.png

**Comment by erdembayar** [comment]
Thank you for asking this question.
Can you help us with repro steps? Can you remove sensitive data/classes/info and share with us bare minimum project/solution which exhibit issue? Without it it’s hard to repro for us.
